test.rs

```diff
@@ -0,0 +1,21 @@
+trait AsMeh {
```

Comment:
This experiment could help with https://webaudio.github.io/web-audio-api/#AudioWorkletProcessor-instantiation